packages/swap/src/routes/__tests__/v2/quote.test.ts

```diff
@@ -53,6 +53,7 @@ jest.mock('../../../pricing/checkPriceWarning', () => ({
 describe(getDcaQuoteParams, () => {
   beforeEach(() => {
     jest.clearAllMocks();
+    jest.restoreAllMocks();
```

Comment:
this didn't even do anything. am i joke to you?

Reply:
i never understood when to use these methods